### ISSUE TITLE
Infinite loop when a jstransformer for html is available

### ISSUE THREAD

**Issue by tomyam1** [metalsmith/metalsmith-in-place#172]
If any of the [HTML jstransoformer](https://github.com/jstransformers/inputformat-to-jstransformer/blob/1.3.3/dictionary.json#L242) are available we'll get to an infinite loop.

Since a template file is compile to HTML and then the HTML template is [re-compiled to HTML](https://github.com/metalsmith/metalsmith-in-place/blob/v4.4.0/lib/index.js#L61) and it goes forever...

For example, pug => html => html => html => ...

Fixed it locally by doing

```typescript
export function requireFromAnotherModule<T>(module: string, fromModule: string): T {
    return require(
        require.resolve(module, {
            paths: [
                require.resolve(fromModule),
            ],
        }),
    )
}

const jstransformersDictionary = requireFromAnotherModule<any>(
        'inputformat-to-jstransformer',
        'metalsmith-in-place',
    ).dictionary as {
        [ext: string]: string[]
    };
jstransformersDictionary.html = [];
```

**Comment by ismay** [comment]
Ah, thanks for notifying me. That's not supposed to happen. Fortunately it's an easy fix. We'll just have to update the render function (here https://github.com/metalsmith/metalsmith-in-place/blob/master/lib/index.js#L11).

I don't think I'll have time for this until Friday though. Feel free to get a PR started in the meantime if you have time.

**Comment by ismay** [comment]
Published as 4.4.1. Let me know if that fixes it for you!

**Comment by tomyam1** [comment]
Thanks @ismay for the quick fix! 
Works fine now
